webpack.config.dev.js

```diff
@@ -22,8 +22,24 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['babel'],
-      include: path.join(__dirname, 'source')
+      include: path.join(__dirname, 'source'),
+      loader: 'babel',
+      query: {
```

Comment:
The problem with query approach - you can't have multiple loaders.

It could be solved in one or the other way, but is very confusing for people.

`.babelrc` is for babel only, it should not affect any other tools. And it should work the same way as with query params. What was the problem with `react-transform`?